### ISSUE TITLE
When typing on google translator, nvda informs the existence of a menu where it is not possible to interact with the resource

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#10756]
### Steps to reproduce:
open the google translator.
start typing some text to translate.
You will hear menu.
Try pressing directional arrows or enter.
### Actual behavior:
the nvda will be mute
### Expected behavior:
this appears to be a word suggestion feature.
When the nvda announces menu there appears to be a list of word suggestions displayed, and with the arrows below you choose a suggestion.
The nvda must have a way for the user to interact with this menu, without the user also being unable to revise the currently inserted text with the arrows.
Or if convenient, nvda can automatically announce suggestions, and the user must have a way to accept the suggestion perhaps by pressing space like it does on smartphone keyboards.
#### NVDA installed/portable/running from source:
portable
#### NVDA version:
alfa 19604
#### Windows version:
10 1909
#### Name and version of other software in use when reproducing the issue:
chrome 79

#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
log:
[tradutor.txt](https://github.com/nvaccess/nvda/files/4153379/tradutor.txt)

**Comment by Adriani90** [comment]
@fernando-jose-silva on NVDA alpha 19604,7773e8ff and Chrome 79 I cannot reproduce this. There are no suggestions displayed and when I am typing there is nothing like "menu" reported.
Could you elaborate more on your chrome settings? Are you using the translator we app? Or the translator website?

**Comment by fernando-jose-silva** [comment]
this is on google online translator

Enviado do meu iPhone

**Comment by Adriani90** [comment]
I can reproduce this in every browser (Chrome 81, Firefox 76 and Edge Chromium 81). It seems NVDA does not recognize the word suggestions and "object replacement character" is reported when pressing enter on a word suggestion.

**Comment by Adriani90** [comment]
@fernando-jose-silva this menu with the suggestions is now working only in Firefox. in Chrome  or Edge there is no menu displayed. Can you test if this issue still occurs for you in Chrome or Edge 112?

**Comment by fernando-jose-silva** [comment]
in chrome it could be some setting, in my chrome the suggestions are shown in the menu and are announced by nvda.
The problem I encountered is that once in the word suggestion list I couldn't close the suggestions on pressing esc, even if I press esc the word suggestion is entered.
I don't know if nv access can do everything in this case or if it's a problem with the chrome interface.

**Comment by Adriani90** [comment]
@fernando-jose-silva how is this on your side with NVDA 2023.2 and Chrome 116? Can you still reproduce this? On my end NVDA does not report any suggestions or menu with suggestions at all. I think this feature with suggestions has been removed from translate if i am not wrong.

**Comment by fernando-jose-silva** [comment]
Thank you very much, I still find nvda's ads very strange in Google Chrome, but that's a subject for new tickets.I can still see the suggestion menus, but nvda now reads them correctly.I believe we can close this call.Enviado do meu iPhoneEm 5 de set. de 2023, à(s) 14:45, Adriani90 ***@***.***> escreveu:﻿
@fernando-jose-silva how is this on your side with NVDA 2023.2 and Chrome 116? Can you still reproduce this? On my end NVDA does not report any suggestions or menu with suggestions at all. I think this feature with suggestions has been removed from translate if i am not wrong.

**Comment by Adriani90** [comment]
Ok thanks, I am closing this as works for me as per comment above.
